assets/styles/style.css

```diff
@@ -401,23 +401,6 @@ PLUGIN STYLES
   background-position: 3px 4px;
 }
 
-@media (max-width:767px) {
```

Comment:
The reason this media query was here is to keep it with all of the other menu item stuff (just above it). Let's keep it here instead of moving it to the bottom of the file.